model/godleyIcon.h

```diff
@@ -40,7 +40,7 @@ namespace minsky
     /// for placement of bank icon within complex
     float flowMargin=0, stockMargin=0;
     /// icon scale is adjusted when Godley icon is resized
-    //float m_iconScale=1;
+    float m_sf=1;
```

Comment:
Why not use Item::m_sf here?

Reply:
I guess I can dynamic_cast to Godley in Item::scaleFactor(const float& sf) to prevent the bounding box from being updated in that case.